Dockerfile.magna

```diff
@@ -0,0 +1,24 @@
+# Development image.
```

Comment:
I'm not familiar with this pattern - so here we define a development image, that we use to building the binary, and then create another docker container that we load the binary into? Cool!

Reply:
That's correct. Basically it's to avoid needing to take all the build dependencies (e.g., the go compiler) into the image that we then run in a bunch of places :-)